pkg/cvo/sync_worker.go

```diff
@@ -189,6 +191,8 @@ type SyncWorker struct {
 	// always be implicitly enabled.
 	// This contributes to whether or not some manifests are included for reconciliation.
 	alwaysEnableCapabilities []configv1.ClusterVersionCapability
+
+	stillInitializingFunc func() bool
```

Comment:
this exists only for tests, right? if so let's tell the reader in a comment

Reply:
Done!